soepy/pre_processing/model_processing.py

```diff
@@ -171,8 +173,7 @@ def flatten_model_spec_dict(model_spec_dict):
     groups = [
         "GENERAL",
         "CONSTANTS",
-        "INITIAL_CONDITIONS",
```

Comment:
Why is this removal possible now?

Reply:
Now the group is called "EDUC", so this is not removed, but rather renamed.